constants/responses.js

```diff
@@ -2,15 +2,60 @@
 
 module.exports = {
   FR: {
+    connectedAs: "Tu es connecté en tant que %s",
+    domainEditDns: "Tu dois modifier la zone DNS de ton domaine afin d'y ajouter le champs A avec l'ip de ton hébergement web, cette ip se trouve sur la page d'informations de ton hébergement dans l'espace client OVH.",
+    disconnected: "Tu n'es pas correctement connecté à ton compte OVH :( , il te suffit de me demander 'connecte moi' pour te reconnecter.",
+    dnsEditDns: "Tu peux modifier tes serveurs via l'espace client OVH",
+    feedbackBadUnderstanding: "Mauvaise comprehension",
+    feedbackHelp: "Est-ce que cette réponse vous a aidé ?",
+    feedbackNo: "Non",
+    feedbackThanks: "Merci pour votre avis",
+    feedbackYes: "Oui",
+    goToManager: "Acceder au manager",
+    hostingSelectSite: "Sélectionne le site concerné",
+    hostingSelectHost: "Sélectionne l'hébergement web sur lequel est installé ton site",
+    hostingNoSite: "Tu n'as pas d'hébergement web :(",
+    hostingWrongSite: "Tu as du sélectionner le mauvais hébergement web associé à ce domaine.",
+    hostingSuspended: "Ton service hébergement web semble être suspendu, pour le réactiver il faut le renouveler via le manager",
+    ndhGetQR: "Obtenir le QR Code",
+    ndhQuestionStart: "Salut à toi 👋, tu as soif ? 🍺 Il faut la mériter si tu en veux. ",
```

Comment:
intended, as in `message/ndh_question.js`, theres is a concatenation with ndhQuestion :)